### PR TITLE
chore(curriculum): add personal portfolio lab

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by moT01** [comment]
Blocking this while we figure out if we want to use the same or new challenge ID's

**Comment by moT01** [comment]
The tests here are failing. It might just need a rebase.

**Review by huyenltnguyen — COMMENTED** [review]
(I was just poking around. Don't count on me for review 😅 )

File: curriculum/challenges/english/25-front-end-development/lab-personal-portfolio/66d98b777c5808d735c47295.md [hunk 1]
```diff
@@ -0,0 +1,661 @@
...
+# --solutions--
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+        integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous" />
```
[the remark this answers is not in the document]
Comment (zairahira) [reply curriculum/challenges/english/25-front-end-development/lab-personal-portfolio/66d98b777c5808d735c47295.md]: Thanks for taking a look @huyenltnguyen, your review is always welcome.
I tested and the updated logo doesn't have an underline now 👍️

File: curriculum/challenges/english/25-front-end-development/lab-personal-portfolio/66d98b777c5808d735c47295.md [hunk 2]
```diff
@@ -0,0 +1,661 @@
...
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. Your portfolio should have a welcome section with an `id` of `welcome-section`.
+2. The welcome section should have an `h1` element that contains text.
+3. Your portfolio should have a projects section with an `id` of `projects`.
```
[the remark this answers is not in the document]
Comment (zairahira) [reply curriculum/challenges/english/25-front-end-development/lab-personal-portfolio/66d98b777c5808d735c47295.md]: I have updated this in the seed code and tests.

**Review by jdwilkin4 — APPROVED** [review]

**Review by moT01 — CHANGES_REQUESTED** [review]
The filename should change to match the challenge id.

**Review by moT01 — APPROVED** [review]

**Pull request merged**